doc_admin/AdminManual.adoc

```diff
@@ -407,11 +403,11 @@ Specify, according to the values specified in the installation phase:
 
 NOTE: These settings are required for every Open Hospital installation/client.
 
-More information on these files can be found in the sections <<database-properties,database.properties>> and
-<<log4j-properties,log4j.properties>>.
+More information on these files can be found in the sections under <<configuration,3. Configuration>> section.
```

Comment:
Isn't the trailing `section` after the reference redundant? 
What about: 
`More information on these files can be found in the <<configuration,3. Configuration>> section.`

Reply:
absolutely, thanks!